static/js/publisher/release/components/revisionsListRow.js

```diff
@@ -69,6 +95,50 @@ const RevisionsListRow = props => {
 
   const showProgressiveReleases = isProgressiveReleaseEnabled && !showChannels;
 
+  const handlePauseProgressiveRelease = () => {
+    releaseRevision(revision, channel, progressiveState);
```

Comment:
Why both `releaseRevision` and `pauseProgressiveRelease` actions are needed to pause? (and resume)?

Reply:
If the release isn't already pending, this just uses the established mechanism to add it to the pendingReleases state.
I guess `pause...`, `resume...` and `...update` could take all the arguments of `releaseRevision` and add it to `pending` if they didn't exist?

Reply:
Yes, would feel like a more intuitive API.
From a component level we don't care if it's already pending or not, just dispatch an action with what we want to pause/update. Reducer should figure out how state needs to be changed.